### ISSUE TITLE
After upgrading to Packer 0.8.0 I get `Invalid value '' for virtualizationType`

### ISSUE THREAD

**Issue by kvz** [mitchellh/packer#2329]
```bash
==> Some builds didn't complete successfully and had errors:
--> tosip-main-ec2: Error registering AMI: InvalidParameterValue: Invalid value '' for virtualizationType.
	status code: 400, request id: []
```

adding:

```json
"ami_virtualization_type": "paravirtual"
```

To my config fixed it for me, but I according to the docs this should have been the default?

**Comment by catsby** [comment]
Hey @kvz – which AMI builder are you using, `amazon-instance` by chance?

**Comment by kvz** [comment]
Yes indeed, am I overlooking something obvious?

**Comment by catsby** [comment]
@kvz oh, no, I think *I* overlooked something :smile: 

I hit this error before using `amazon-chroot` and didn't fix the issue for `amazon-instance` as well. I'm fixing that now though ...

**Comment by catsby** [comment]
https://github.com/mitchellh/packer/pull/2330 should fix this, but I don't have an `amazon-instance` config file handy so I'm working on verifying. Are you able to build that branch of Packer and try it out?

**Comment by kvz** [comment]
I'd like to help but we're closing in on the weekend here in Amsterdam and I still have a few other things on my plate I'd like to finish! :smile:

**Comment by catsby** [comment]
@kvz no worries, you've done enough :smile: thanks for reporting this and the other issue as well!

**Comment by catsby** [comment]
Merged #2330, thanks again!
